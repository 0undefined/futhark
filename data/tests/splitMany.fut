-- ==
-- input {
-- }
-- output {
--   [1]
--   empty(int)
--   [2, 3]
--   [4]
-- }
fun main(): ([]int,[]int,[]int,[]int) =
    let a = [1,2,3,4] in
    split( (1,1,3), [1,2,3,4] )
