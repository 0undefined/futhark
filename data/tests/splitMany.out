[1]
empty(int)
[2, 3]
[4]
